internal/secrets/secret_test.go

```diff
@@ -50,7 +50,7 @@ func TestFetchSecrets_Success(t *testing.T) {
 	})
 
 	keys := []string{"DATABASE_URL", "API_TOKEN"}
-	secrets, errs := FetchSecrets(context.Background(), apiClient, "test-job-id", keys, false)
+	secrets, errs := FetchSecrets(context.Background(), apiClient, "test-job-id", keys, 1)
```

Comment:
While you're here, why not update all these:

```suggestion
	secrets, errs := FetchSecrets(t.Context(), apiClient, "test-job-id", keys, 1)
```